client/src/components/PacmanGame/gameCore.js

```diff
@@ -1,19 +1,13 @@
-export const initGridState = (rows, cols) => {
-  // const column = new Array(cols).fill(0);
-  // return new Array(rows).fill(column);
-  return Array(rows).fill().map(() => Array(cols).fill(0))
-};
-
-
-
-
+export // const column = new Array(cols).fill(0);
+// return new Array(rows).fill(column);
+const initGridState = (rows, cols) => Array(rows).fill().map(() => Array(cols).fill(0));
```

Comment:
why fill is empty?

Reply:
@nimish-gupta The `fill` is there to put `undefined` for all array positions so that `map` can run. This is the fastest way to create 0 filled 2d array.

@kaushikb11 I don't see the benefit of using the `Array(rows).fill(Array(columns).fill(0)).map(arr => arr.slice())` over current implementation.